docs/getting_started/installation_guide/platforms/karamel.rst

```diff
@@ -0,0 +1,85 @@
+.. _karamel-installer:
+
+=======================
+Karamel
+=======================
+
+#. Download and install `Karamel <http://www.karamel.io/>`_ 
+#. Run Karamel.
+#. Click on the "Load Cluster Definition" menu item in Karamel. You are now prompted to select a cluster definition YAML file.
+
+Example cluster definitions can be found on `github <https://github.com/logicalclocks/karamel-chef/tree/master/cluster-defns>`_.
+
+
+**Choosing which services to run on which nodes**
+
+You need to decide which services you will install on which nodes. In Karamel, we design a set of *Node Groups*, where each *Node Group* defines a stack of services to be installed on a machine. Each machine will only have one *Node Group* set of services.
+We now provide two recommended setup:
```

Comment:
Fixed. I don't like this whole section. Maybe we should remove it? What do you think?

Reply:
I think we should at least make the section more concrete with example karamel specs for the recommended setups. Right now it is a bit high level and it isn't intuitive on how you would install the recommended setups.

Reply:
Honestly, we don't support karamel as a way to install hopsworks, and we don't intend to. So, i won't make an effort to write a bunch of example cluster definitions. It will create too many support issues for people who only want to use community hopsworks. Remove or leave this section as is?